chart/values.yaml

```diff
@@ -273,8 +273,10 @@ io_engine:
   envcontext: ""
   reactorFreezeDetection:
     enabled: false
-  # -- The number of cpu that each io-engine instance will bind to.
+  # -- The number of cores that each io-engine instance will bind to.
   cpuCount: "2"
+  # -- Overrides the cpuCount and explicitly sets the list of cores.
+  coreList: []
```

Comment:
I assume the override happens only if the list is non-empty, right?

Reply:
Yes correct